.gitignore

```diff
@@ -27,27 +27,17 @@ ENV/
 *.swo
 *~
 
-# System files
-# Ignores system-specific files like .DS_Store (macOS) and Thumbs.db (Windows).
-.DS_Store
-Thumbs.db
 
 # Docker
 # Ignores Docker-related files and directories to prevent accidental inclusion or modification.
-docker-compose*.yml
-Dockerfile
-*.dockerignore
+
 *.env
 .env.*
 !example.env
 
 # Databases
 # Ignores database files and directories, typically for SQLite.
 database/
-*.db
-*.sql
-*.sqlite
-*.sqlite3
 
```

Comment:
**issue:** we don't wannt `*.{db,sqlite,sqlite3}` files in here, those are binary formats that shouldn't be in the repo - `*.sql` removal is fine. Please add instructions how to create this db instead